.github/checkout-build-openapi/action.yml

```diff
@@ -11,16 +16,24 @@ runs:
         java-version: '17'
         cache: 'maven'
 
-    - name: Checkout OpenAPI repository
+    - name: Get openapi version from pom
+      id: get_openapi_version
+      run: |
+        VERSION=$(mvn help:evaluate -Dexpression=payforlegalaid-openapi.version -q -DforceStdout)
+        echo "version=$VERSION" >> "$GITHUB_OUTPUT"
```

Comment:
@RobertBuczek  sent me a message this morning about the risks with Log4j. What do you think and is there a way to alleviate it?